### PR TITLE
Update ObservableList to Pair class

### PR DESCRIPTION
**Pull request by Jaspertzx**
Currently, the ObservableList for Orders only
contains Orders.

This will not allow any names to show up for
orders, which may make the user confused who's
order is this.

Update the ObservableList to hold a Pair class.

This will allow us to display both Person and
Order details.

### CODE REVIEW

**Comment by Jaspertzx** [comment]
Resolves #108

File: src/main/java/seedu/address/ui/OrderCard.java [hunk 1]
```diff
@@ -33,6 +36,8 @@ public class OrderCard extends UiPart<Region> {
     @FXML
     private Label orderId;
     @FXML
+    private Label orderCustomerName;
```
Comment (kailinteoo) [inline src/main/java/seedu/address/ui/OrderCard.java:39]: Do stick with the naming convention, and change orderCustomerName to orderClientName

**Review by kailinteoo — COMMENTED** [review]
other then that, LGTM

File: src/main/java/seedu/address/ui/OrderCard.java (hunk 1 above)
Comment (Jaspertzx) [reply src/main/java/seedu/address/ui/OrderCard.java:39]: Understood, thank you for catching this error. I will make the amendments immediately.

File: src/main/java/seedu/address/logic/commands/orders/DeleteOrderCommand.java [hunk 2]
```diff
@@ -50,38 +50,28 @@ public DeleteOrderCommand(Index targetIndex) {
...
-                Person editedPerson = new Person(
-                        person.getName(), person.getPhone(), person.getEmail(),
-                        person.getAddress(), person.getTags(),
-                        removeOrder(orderToDelete, person.getOrders()));
-
-                return new Pair<>(person, editedPerson);
-            }
-        }
-        throw new CommandException(MESSAGE_DELETE_ORDER_FAILURE);
+    private Person getEditedPerson(Person person, Order orderToDelete) {
```
Comment (rertyy) [inline src/main/java/seedu/address/logic/commands/orders/DeleteOrderCommand.java:71]: Might need to change this name when editOrder comes in

**Review by rertyy — APPROVED** [review]

**Pull request merged**